addons/mail/static/src/models/attachment_list.js

```diff
@@ -71,6 +71,66 @@ registerModel({
         _computeViewableAttachments() {
             return replace(this.attachments.filter(attachment => attachment.isViewable));
         },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsDiscuss() {
+            return Boolean(
+                (this.messageViewOwner && this.messageViewOwner.isDiscuss) ||
+                (this.composerViewOwner && this.composerViewOwner.isDiscuss)
+            );
+        },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsChatWindow() {
+            return Boolean(
+                (this.messageViewOwner && this.messageViewOwner.isChatWindow) ||
+                (this.composerViewOwner && this.composerViewOwner.isChatWindow)
+            );
+        },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsChatter() {
+            return Boolean(
+                (this.messageViewOwner && this.messageViewOwner.isChatter) ||
+                (this.composerViewOwner && this.composerViewOwner.isChatter)
+            );
+        },
```

Comment:
Missing `in` in-between `is` and `Discuss/ChatWindow/Chatter`
An Attachment List is no a X, but is in a X